core.go

```diff
@@ -18,6 +18,44 @@ const (
 	zapSentryScopeKey = "_zapsentry_scope_"
 )
 
+var (
+	defaultFrameFilters = FrameFilters{
+		SkipModulePrefixFrameFilter("github.com/TheZeroSlave/zapsentry"),
+		SkipFunctionPrefixFrameFilter("go.uber.org/zap"),
+	}
+)
+
+type (
+	FrameFilter interface {
+		SkipFrame(f sentry.Frame) bool
```

Comment:
I don't like this name ... I think I'd prefer `Filters` or something similar.

Reply:
tbh I don't like `Filters` either :sweat_smile: how about:

```go
type FrameSkipper interface {
    Skip(f sentry.Frame) bool
}
```

to honor the interface name <-> function name similarity 'pattern' but `Filters` doesn't really fit, does it?

Reply:
Well, I don't like that either, and let me tell you why: when I look at just the interface, I would guess that the implementation is expected to skip the frame (somehow, whatever that means) and return the result of such operation. Which obviously isn't what we want implementations to do here.

But hey, maybe it's just me. I'm also not a native English speaker, so that might influence my feeling about it a bit too...

Reply:
I actually don't see a problem with being "java-like", because many Java patterns are good (and many are, of course, horrible). I actually tend to write Go more like I would write PHP, Java and other similar languages and it works quite well. I don't like the Go philosophy to try to write every single variable name as short as possible :D 

The `FrameSkipPredicate` truly sounds like a java port :rofl: But yeah, I would approve that as it IMO can't be mistaken for anything else, and it covers the functionality perfectly. `FrameMatcher` or `FrameFilterMatcher` would be more in line with Go style though. Anyway, now I'm just being too pedantic :D What matters most is that the functionality makes sense and is a good addition :-)

Reply:
I actually already thought of something like a `Matcher` but I used that term so often in the last time that I figured something else would be nice :smile: but I guess it just makes sense most so I'll take that and update the code tomorrow.

Thanks for the input and the patience! :blush: